clang/test/CodeGen/unified-lto-pipeline-ps.c

```diff
@@ -0,0 +1,33 @@
+// RUN: %clang -flto=thin --target=x86_64-scei-ps4 -O2 -c %s -o %t.ps4.tu -Xclang -fdebug-pass-manager 2>%t.ps4.tu.txt
```

Comment:
unified-lto-pipeline.c is present and serves the purpose. For target-generic codegen features, we usually test just one platform (it doesn't really scale if every platform that does no customization wants to add their own tests). If PS ever customizes the behavior, we can accept the new test.